Kitodo/src/main/java/org/kitodo/production/helper/tasks/NewspaperMigrationTask.java

```diff
@@ -72,7 +72,7 @@
      * @param transferredBatch
      *            an object that transfers a newspaper batch from index
      */
-    public NewspaperMigrationTask(BatchDTO transferredBatch) {
+    public NewspaperMigrationTask(Batch transferredBatch) {
```

Comment:
If you change batch transfer object (`BatchDTO`) to `Batch`, you could rename the variable `transferredBatch` to `batch`, too. Just if you want. Same for the field